src/librustdoc/html/templates/source.html

```diff
@@ -1,5 +1,7 @@
 <div class="example-wrap"> {# #}
-    <pre class="src-line-numbers">
+    {# https://developers.google.com/search/docs/crawling-indexing/robots-meta-tag#data-nosnippet-attr
+       Do not show "1 2 3 4 5 ..." in web search results. #}
+    <pre class="src-line-numbers" data-nosnippet>
```

Comment:
>  This can be done on an HTML-element level with the data-nosnippet HTML attribute on span, div, and section elements. 

`pre` isn't in the list where that's supported.